packages/block-library/src/gallery/edit.js

```diff
@@ -293,6 +296,14 @@ function GalleryEdit( props ) {
 					newOrderMap[ b.attributes.id ]
 			)
 		);
+
+		//If new blocks added select the first of these so they scroll into view.
```

Comment:
Tiniest of nits: add a whitespace character after the `//` — I wouldn't worry about this unless you're making other changes though!
```suggestion
		// If new blocks added, select them so they scroll into view.
```

Reply:
Fixed.